### DISCUSSION TITLE
Weekly Sync - 13 June 2022

### DISCUSSION THREAD

**Discussion by matdehaast** [interledger/rafiki#369] [General]
## Updates 

- Nathan
   - Working on interaction step
   - Stuck on test due to the koa mock having issue with cookies.
   - Will move onto the continue grant endpoint.
- Sabine
  - Working on Token Introspection endpoint, merged now
  - Working on OP
    - Added connections endpoint to cater for WM.
    - Need to change `ilp-pay` to account for the change in the shape
- Alex
  - Get written offer out to Roman. Still waiting for information
  - Focusing on Rafiki docs.
- Damien
  - Token revocation PR is up.
  - Add signature verification to AS
- Matt
   - Adding issues from Fynbos Integration
   - Put high-level vision for Rafiki Summit

## Timelines & Project Scope
[ Main milestones](https://github.com/interledger/rafiki/milestones)

**Comment by A-Mozeak** [comment]
Hey thanks again for having us! Just wanted to point out that the page at https://github.com/interledger/rafiki/pkgs/container/rafiki-backend responds with a 404.
